lib/svg-sprite/constants.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    DEFAULT_XML_DECLARATION: '<?xml version="1.0" encoding="utf-8"?>'
```

Comment:
This seems useless just for one case IMHO.

Reply:
Maybe, but at the same time it adds more lucidity where it used.

Reply:
I'd still drop it. Unless, the file has a clear reason of existence, better drop it.

Reply:
Ok